src/ORM/RulesChecker.php

```diff
@@ -113,6 +114,26 @@ public function existsIn($field, $table, $message = null)
     }
 
     /**
+     * Returns a callable that can be used as a rule for checking that the entity is not
+     * referenced by any row in the specified table
+     *
+     * This is useful for enforcing foreign key integrity checks. (i.e.: ON DELETE RESTRICT)
+     *
+     * ### Example:
+     *
+     * ```
+     * $rules->add($rules->isNotReferencedBy('Articles'));
+     * ```
+     *
+     * @param string $table The table name to check for references
+     * @return callable
+     */
+    public function isNotReferencedBy($table)
```

Comment:
@robertpustulka I was using `public function existsIn($field, $table, $message = null)` as a base for mine. There it is called a table, and what `isNotReferencedBy` does, is very similar, just in reverse. I could rename it, but then wouldn't this need renaming too?